tests_integration/nativeapp/test_bundle.py

```diff
@@ -23,13 +23,27 @@
 )
 
 
-@pytest.fixture(scope="function", params=["napp_init_v1", "napp_init_v2"])
+@pytest.fixture(
+    scope="function",
+    params=[
+        ["app bundle", "napp_init_v1"],
+        ["app bundle", "napp_init_v2"],
+        ["ws bundle --entity-id=pkg", "napp_init_v2"],
+    ],
+)
 def template_setup(runner, project_directory, request):
-    test_project = request.param
+    command, test_project = request.param
     with enable_definition_v2_feature_flag:
         with project_directory(test_project) as project_root:
             # Vanilla bundle on the unmodified template
-            result = runner.invoke_json(["app", "bundle"])
+            if command.startswith("ws"):
+                execute_bundle_command = lambda: runner.invoke_with_connection(
+                    command.split()
+                )
+            else:
+                execute_bundle_command = lambda: runner.invoke_json(command.split())
+            result = execute_bundle_command()
```

Comment:
why the lambda? how about
```suggestion
            if command.startswith("ws"):
                result = runner.invoke_with_connection(command.split())
            else:
                result = runner.invoke_json(command.split())
```

Reply:
Passing the lambda to the tests to be able to execute the bundle command a second time (see other comment below)